src/Simplex/Messaging/Agent/Store/SQLite/Migrations/M20230320_retry_state.hs

```diff
@@ -0,0 +1,13 @@
+{-# LANGUAGE QuasiQuotes #-}
+
+module Simplex.Messaging.Agent.Store.SQLite.Migrations.M20230320_retry_state where
+
+import Database.SQLite.Simple (Query)
+import Database.SQLite.Simple.QQ (sql)
+
+m20230320_retry_state :: Query
+m20230320_retry_state =
+  [sql|
+ALTER TABLE snd_messages ADD COLUMN retry_int_slow INTEGER NULL;
+ALTER TABLE snd_messages ADD COLUMN retry_int_fast INTEGER NULL;
```

Comment:
I think convention of the existing code base is that nullability is not explicitly specified, why change?
```suggestion
ALTER TABLE snd_messages ADD COLUMN retry_int_slow INTEGER;
ALTER TABLE snd_messages ADD COLUMN retry_int_fast INTEGER;
```